executor.go

```diff
@@ -223,15 +224,30 @@ func (e *Executor) Wait(ctx context.Context) (*ExecResult, error) {
 	}
 
 	if e.capture {
-		res.Stdout = string(e.stdoutBuf.Bytes())
-		res.Stderr = string(e.stderrBuf.Bytes())
+		res.Stdout = e.stdoutBuf.String()
+		res.Stderr = e.stderrBuf.String()
 	}
 
 	res.Runtime = e.TimeRunning()
 
 	return res, err
 }
 
+func (e *Executor) flush(bufType string) {
+	r := e.stdout
+	w := &e.stdoutBuf
+	if bufType == "stderr" {
```

Comment:
ah, yeah good catch. I was trying to do something else before with this string. Let me make the change.